graph/db/graph.go

```diff
@@ -1,26 +1,93 @@
 package graphdb
 
-import "github.com/lightningnetwork/lnd/kvdb"
+import (
+	"time"
+
+	"github.com/lightningnetwork/lnd/graph/db/models"
+	"github.com/lightningnetwork/lnd/kvdb"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/routing/route"
+)
+
+// Config is a struct that holds all the necessary dependencies for a
+// ChannelGraph.
+type Config struct {
```

Comment:
nit: `KVDBConfig`? No strong opinion here tho, as this is a place I find myself struggling sometimes. It's nice to just name it `Config`, as suggested by the official go, and use it as `graphdb.Config`. But in practice it's easier to search using a unique name.

Reply:
so this is the config struct for ChannelGraph and not for the KVStore. It just at the moment happens to only hold KVStore related things but will be expanded in future.